Do not install package-lock.json into the wheel
boo#1231254 gh#openSUSE/obs-service-node_modules#41

diff -ur panel-1.5.0.orig/pyproject.toml panel-1.5.0/pyproject.toml
--- panel-1.5.0.orig/pyproject.toml	2024-10-03 11:00:09.362861013 +0200
+++ panel-1.5.0/pyproject.toml	2024-10-03 11:00:21.459903574 +0200
@@ -110,6 +110,7 @@
 
 [tool.hatch.build.targets.wheel]
 include = ["panel"]
+exclude = ["panel/package-lock.json"]
 
 [tool.hatch.build.targets.wheel.force-include]
 "panel/dist" = "panel/dist"
